lib/jxl/splines.h

```diff
@@ -79,6 +79,16 @@ class QuantizedSpline {
   int sigma_dct_[32] = {};
 };
 
+struct SplineSegment {
```

Comment:
Roughly what is it that we call a segment here (for example why does it only have an x range)? Maybe a short explanatory comment wouldn’t hurt.

My understanding from the code is that it represents the intersection between a row in the image and the square of one Gaussian splat. Is that correct?

Reply:
Yup, added a comment.

Reply:
I think “line” -> “row” might be slightly less ambiguous. What do you think?